src/App.js

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
 import './App.css';
-// eslint-disable-next-line import/extensions
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Home from './components/Home';
 import Calculator from './components/Calculator';
+import Quote from './components/Quote';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <Calculator />
-        </header>
-      </div>
-    );
-  }
-}
+const App = () => (
+  <div className="App">
+    <Header />
+    <Routes>
+      <Route path="Quote" element={<Quote />} />
+      <Route path="calculator" element={<Calculator />} />
+      <Route path="Home" element={<Home />} />
+    </Routes>
```

Comment:
- [ ] The homepage path is supposed to be the root path, Please make the path `/` instead of `Home` because now the homepage is a blank page.